### DISCUSSION TITLE
no skipping or pauze

### DISCUSSION THREAD

**Discussion by Tecko71** [LMS-Community/slimserver#932] [General]
I did notice dat you can not skip and resume songs in LMS you have to start over or go to next song. It is whit streaming service and it is with local library.

**Comment by mherger** [comment]
What music services? What type of local file? What player? 

Seeking, pausing etc. certainly does work in most cases. There are a few limitations with certain online services. But in general this should work as expected.

**Comment by Tecko71** [comment]
With deezer plugin it is not working. For example I can pauze or seek. After that I have to restart the song again. Oh I test it it is only with Deezer. In local library it works fine. And I use squeezelite-x.
